<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMCBinSettingClass</class>
 <widget class="QWidget" name="DialogMCBinSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogMCBinSetting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#E0E0E0;
font-family:Microsoft YaHei;
background: #323232;




</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;
}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QLineEdit
{
	color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;
}

</string>
     </property>
     <property name="title">
      <string>Connection Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_IpStr"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Port"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Address:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Addres"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_Connet">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_DisConnect">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;
}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
     </property>
     <property name="title">
      <string>Reading Data Test</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTextEdit" name="textEdit_Read">
          <property name="styleSheet">
           <string notr="true">color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_IntRead">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Int Read</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_DoubleRead">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Float Read</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_StringRead">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>String Read</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;
}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
     </property>
     <property name="title">
      <string>Writting Data Test</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextEdit" name="textEdit_Write">
          <property name="styleSheet">
           <string notr="true">color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,1,0,1,0,1,0">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit
{
	color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;
}</string>
            </property>
            <property name="title">
             <string>Input Data</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLineEdit" name="lineEdit_InputData"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_IntWrite">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Int Write</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_DoubleWrite">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Float Write</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_StringWrite">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>String Write</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
